ui.js

```diff
@@ -0,0 +1,67 @@
+// Called after a WIN or a LOSS.
+// Intended to disconnect the user interface from the game state, 
+// freezing the html and preventing the user from playing the game
+// any further.
+// User would be forced to either quit or refresh the page for
+// a new game.
+var stopGame = function () {
+	// $("script:eq(1)").remove();	//#1
+
+	//$("#USER_INTERFACE").remove();	//#2
+	
+	//$(document).replace("script", "NOT_A_SCRIPT");	//#3
+
+	// var replaceLoc = $(document.head);	//#4
+	// var regExMatch = "script";
+	// var replaceWith = "NOT_A_SCRIPT";
+	// var result = replaceLoc.replace(regExMatch, replaceWith);
+	// $("head").html(result);
+};
+
+
+$(document).ready(function() {
+	document.oncontextmenu = function() {return false;};
```

Comment:
This will prevent right clicking anywhere on the screen. You only want to change its behavior for the cells. Let's remove this and handle the event's behavior in the mousedown handler.